src/main/java/com/exasol/adapter/dialects/AbstractSqlDialect.java

```diff
@@ -184,7 +184,10 @@ public void validateProperties() throws PropertyValidationException {
         validateSchemaNameProperty();
         validateDebugOutputAddress();
         validateExceptionHandling();
-        validateDataTypeDetectionStrategy();
+        PropertyValidator.chain() //
+                .add(DataTypeDetection.getValidator()) //
+                .add(TableCountLimit.getValidator()) //
```

Comment:
Not sure, but would it make sense to put the chain generation into the constructor? That way each Adapter/Dialect could simply add its own validators there, and the validate method could be `final`

Reply:
I agree, but was too lazy until now 😃 I will update the PR.